.github/workflows/agw.branch_push.yml

```
@@ -4,15 +4,20 @@ on:
     branches:
       - "**"
       - "!main"
+      - "!1.6"
 
 jobs:
   agw-lint-report:
     name: Lint report
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - name: Set Environment Variables
+        uses: tw3lveparsecs/github-actions-setvars@latest
```

I know it's netpicking 
```suggestion
        uses: tw3lveparsecs/github-actions-setvars@v0.2
```